src/openrct2/network/network.cpp

```diff
@@ -534,17 +534,23 @@ const char* Network::FormatChat(NetworkPlayer* fromplayer, const char* text)
 	static char formatted[1024];
 	char* lineCh = formatted;
 	formatted[0] = 0;
+	time_t timer;
```

Comment:
This is wasted space & time in case `!fromplayer == true`. Can you move it all down to the conditional block?